modules/creating-an-infra-node.adoc

```diff
@@ -35,11 +35,37 @@ $ oc label node <node-name> node-role.kubernetes.io/infra=""
 $ oc get nodes
 ----
 
-. Create a default node selector so that pods without a node selector are assigned a subset of nodes to be deployed on, for example by default deployment in worker nodes. As an example, the `defaultNodeSelector` to deploy pods on worker nodes by default would look like:
+. Create a default cluster-wide node selector. The default node selector is applied to pods created in all namespaces. This creates an intersection with any existing node selectors on a pod, which additionally constrains the pod's selector.
++
+[IMPORTANT]
+====
+If the default node selector key conflicts with the key of a pod's label, then the default node selector is not applied.
+
+However, be careful not to set a default node selector that will cause a situation where a pod becomes unschedulable. For example, setting the default node selector to a specific node role, such as `node-role.kubernetes.io/infra=""`, when a pod's label is set to a different node role, such as `node-role.kubernetes.io/master=""`, can cause the pod to become unschedulable. For this reason, it is not recommended to set the default node selector to specific node roles.
```

Comment:
Not sure how you feel about shortening the first sentence, but perhaps it is more direct to say something like, "However, do not to set a default node selector that causes a pod to become unschedulable" (or "that can cause"). 🤷